### PR TITLE
Create product-ownership-areas.md

### PR DESCRIPTION
**Pull request by wiersgallak**
PM Areas of Ownership

### CODE REVIEW

File: operations/research-and-development/product/product-ownership-areas.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+| :--- | :--- | :--- |
+| Administration | System Console, Compliance, Authentication, User and Team Management, Permissions| Adam Clarkson |
+| Apps & Integrations | Plugins, Apps Marketplace, Apps & Plugin Framework | Aaron Rothschild |
+| Customer Journey (On-Prem) | Trials, Purchases/Customer Portal, In-product notices, Admin Adivsor, Licenses | Jason Blais |
+| Customer Journey (Cloud) | Trials, Billing, Customer Portal, In-product notices, Admin Adivsor | Katie Wiersgalla | 
+| End-User Experience | Chat, Search, Notifications, Status, User Settings, Onboarding, Mobile | Eric Sethna |
+| Incident Collaboration | Playbooks, Incident Response Workflow | Ian Tao |
+| Platform | High Availabilty, Database, File Storage, Security, Logging, VPC Peering, Kubernetes, Omnibus | Katie Wiersgalla|
+
+For a comprehensive list of ownership areas, please see [this document](https://docs.google.com/spreadsheets/d/1-EUPdTpOxNx4bwf-312Pgqis7xSObK00c-KYwPXr2Y0/edit?usp=sharing).
```
Comment (jasonblais) [inline operations/research-and-development/product/product-ownership-areas.md:12]: Spreadsheet opens in "Sheet 3" tab which looks incomplete, do we want to open in the "Internal" tab?

File: operations/research-and-development/product/product-ownership-areas.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Product Management Areas of Ownership
+
+| Product Area | Example Features| PM Owner |
+| :--- | :--- | :--- |
+| Administration | System Console, Compliance, Authentication, User and Team Management, Permissions| Adam Clarkson |
+| Apps & Integrations | Plugins, Apps Marketplace, Apps & Plugin Framework | Aaron Rothschild |
+| Customer Journey (On-Prem) | Trials, Purchases/Customer Portal, In-product notices, Admin Adivsor, Licenses | Jason Blais |
```
Comment (jasonblais) [inline operations/research-and-development/product/product-ownership-areas.md]: Propose we combine customer journey on-prem + cloud into one row and note ownership breakdown in PM Owner column, like below.

(For an example, it took me a moment to understand why `Trials` showed up twice and who owned it)

Comment (jasonblais) [reply operations/research-and-development/product/product-ownership-areas.md]:
```suggestion
| Customer Journey | Trials, Billing, Purchases/Customer Portal, In-product notices, Admin Advisor, Licenses | Jason Blais (Self-Managed), Katie Wiersgalla (Cloud) |
```

**Review by jasonblais — CHANGES_REQUESTED** [review]

File: operations/research-and-development/product/product-ownership-areas.md (hunk 1 above)
Comment (wiersgallak) [reply operations/research-and-development/product/product-ownership-areas.md:12]: Good catch - removed my working sheet.

**Review by jasonblais — DISMISSED** [review]

File: operations/research-and-development/product/product-ownership-areas.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# Product Management Areas of Ownership
+
+| Product Area | Example Features| PM Owner |
+| :--- | :--- | :--- |
+| Administration | System Console, Compliance, Authentication, User and Team Management, Permissions| Adam Clarkson |
+| Apps & Integrations | Plugins, Apps Marketplace, Apps & Plugin Framework | Aaron Rothschild |
+| Customer Journey | Trials, Billing, Purchases/Customer Portal, In-product notices, Admin Advisor, Licenses | Jason Blais (Self-Managed), Katie Wiersgalla (Cloud) |
```
Comment (cwarnermm) [inline operations/research-and-development/product/product-ownership-areas.md]:
```suggestion
| Customer Journey | Trials, Billing, Purchases/Customer Portal, In-Product Notices, Admin Advisor, Licenses | Jason Blais (Self-Managed), Katie Wiersgalla (Cloud) |
```

File: operations/research-and-development/product/product-ownership-areas.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
+# Product Management Areas of Ownership
+
+| Product Area | Example Features| PM Owner |
+| :--- | :--- | :--- |
+| Administration | System Console, Compliance, Authentication, User and Team Management, Permissions| Adam Clarkson |
```
Comment (cwarnermm) [inline operations/research-and-development/product/product-ownership-areas.md]:
```suggestion
| Administration | System Console, Compliance, Authentication, User and Team Management, Permissions | Adam Clarkson |
```

File: operations/research-and-development/product/product-ownership-areas.md [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+| Apps & Integrations | Plugins, Apps Marketplace, Apps & Plugin Framework | Aaron Rothschild |
+| Customer Journey | Trials, Billing, Purchases/Customer Portal, In-product notices, Admin Advisor, Licenses | Jason Blais (Self-Managed), Katie Wiersgalla (Cloud) |
+| End-User Experience | Chat, Search, Notifications, Status, User Settings, Onboarding, Mobile | Eric Sethna |
+| Incident Collaboration | Playbooks, Incident Response Workflow | Ian Tao |
+| Platform | High Availabilty, Database, File Storage, Security, Logging, VPC Peering, Kubernetes, Omnibus | Katie Wiersgalla|
```
Comment (cwarnermm) [inline operations/research-and-development/product/product-ownership-areas.md]:
```suggestion
| Platform | High Availabilty, Database, File Storage, Security, Logging, VPC Peering, Kubernetes, Omnibus | Katie Wiersgalla |
```

**Review by cwarnermm — COMMENTED** [review]
Non-blocking suggestions provided inline

**Review by cwarnermm — APPROVED** [review]
Looks great and reads well!

**Comment by wiersgallak** [comment]
@jasonblais are you able to merge this PR?  I don't have access.

**Pull request merged**